assets/js/player.js

```diff
@@ -112,6 +112,38 @@ function addCurrentTimeToURL(url) {
     return urlUsed;
 }
 
+/**
+ * Timer that updates the timestamp on all external links
```

Comment:
VideoJS has a `timeupdate` event that fires each time the playback position changes. It's probably a better idea to listen for that instead of using a timer.